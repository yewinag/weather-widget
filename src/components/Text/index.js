import styled from 'styled-components';

const Text = styled.p`
    font-size: 14px;
    margin: 4px;
`
export default Text;