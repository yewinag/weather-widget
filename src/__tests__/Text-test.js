import React from 'react';

import renderer from 'react-test-renderer';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

import Text from '../components/Text';

Enzyme.configure({ adapter: new Adapter() });

describe('P tag', () => {
    it('render p', () => {
        const wrapper = shallow(<Text />)
        expect(wrapper.exists()).toBe(true);
        expect(wrapper).toMatchSnapshot();
    });

});