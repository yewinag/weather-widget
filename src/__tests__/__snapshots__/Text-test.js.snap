// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`P tag render p 1`] = `ShallowWrapper {}`;
